content/usgs-packages/dataRetrieval_readNWIS.Rmd

```diff
@@ -47,15 +47,28 @@ Description <- c(
   "Annual maximum instantaneous streamflows and gage heights.",
   "Discrete water quality data.",
   "Rating table information for active stream gages",
-  "Site metadata information. SHOULD THIS BE HERE?",
+  "Site metadata information",
   "Daily, monthly, or annual statistics for time-series data. Default is mean daily.",
   "Data from the USGS National Water Use Program.",
   "Returns time-series data reported from the USGS Instantaneous Values Web Service."
 )
 Arguments <- unlist(lapply(Function, function(f) { paste(names(formals(f)), collapse = ", ") }))
 
+Function_fancy <- c('<a href="#readnwisdata">readNWISdata</a>', 
+                    '<a href="#readnwisdv">readNWISdv</a>', 
+                    '<a href="#readnwisgwl">readNWISgwl</a>', 
+                    '<a href="#readnwismeas">readNWISmeas</a>',
+                    '<a href="#readnwispcode">readNWISpCode</a>',
+                    '<a href="#readnwispeak">readNWISpeak</a>',
+                    '<a href="#readnwisqw">readNWISqw</a>',
+                    '<a href="#readnwisrating">readNWISrating</a>',
+                    '<a href="#readnwissite">readNWISsite</a>',
+                    '<a href="#readnwisstat">readNWISstat</a>',
+                    '<a href="#readnwisuse">readNWISuse</a>',
+                    '<a href="#readnwisuv">readNWISuv</a>')
+
 library(htmlTable)
-htmlTable(data.frame(Function, Description, Arguments), caption="Table 1. readNWIS function definitions",
+htmlTable(data.frame(Function_fancy, Description, Arguments), caption="Table 1. readNWIS function definitions",
```

Comment:
do these links work? I had to do `[link words](actual link)` and add `results='asis'` to my code chunk to get links in the other tables to work.